common/djangoapps/entitlements/models.py

```diff
@@ -1,15 +1,17 @@
 import uuid as uuid_tools
 from datetime import timedelta
-from util.date_utils import strftime_localized
 
 from django.conf import settings
 from django.contrib.sites.models import Site
 from django.db import models
 from django.utils.timezone import now
+from model_utils.models import TimeStampedModel
 
 from lms.djangoapps.certificates.models import GeneratedCertificate
-from model_utils.models import TimeStampedModel
+from openedx.core.djangoapps.catalog.utils import get_course_uuid_for_course
 from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.models import CourseEnrollment
+from util.date_utils import strftime_localized
```

Comment:
why is this getting pulled in?

Reply:
This was just a sorting of the imports.

Reply:
It looks like my auto sort push if down to here.

Reply:
Gotchya 👍